person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    of_age? || @parent_permission ? true : false
```

Comment:
- [ ] [OPTIONAL] you don't need to include the ternary operator in this case, the below will also return a boolean value:
```
def can_use_services?
  of_age? || @parent_permission
end
```

Reply:
O okay. Noted.
Thanks.